src/main/navigation/navigation.h

```diff
@@ -155,6 +155,7 @@ typedef struct navConfig_s {
         uint16_t launch_throttle;            // Launch throttle
         uint16_t launch_motor_timer;         // Time to wait before setting launch_throttle (ms)
         uint16_t launch_motor_spinup_time;   // Time to speed-up motors from idle to launch_throttle (ESC desync prevention)
+	uint16_t launch_min_time;	     // Minimum time in launch mode to prevent possible bump of the sticks from leaving launch mode early
```

Comment:
Please fix the indentation

Reply:
Done. I need to change my auto-indentation settings for iNav.